test/probes/isUnsafeCommand.spec.js

```diff
@@ -0,0 +1,130 @@
+// Import Node.js Dependencies
+import { test } from "node:test";
+import assert from "node:assert";
```

Comment:
```suggestion
import assert from "node:assert/strict";
```
Also `strict` here :p 
Everything else lgtm